eq-author/src/index.js

```diff
@@ -43,11 +43,20 @@ const history = createHistory();
 
 const httpLink = createHttpLink(config.REACT_APP_API_URL);
 
+const versionLink = setContext((_, { headers }) => ({
```

Comment:
This might be worth extracting as well so we can test it? Maybe create a middleware folder so the questionnaire id and auth headers are handled there as well?